pythoncz/models/github.py

```diff
@@ -81,6 +71,27 @@ def _get_issues_for_org(session, org_name):
         )
 
 
+def _request_api(session, query, variables):
+    res = session.post('https://api.github.com/graphql', json={
+        'query': query,
+        'variables': variables,
+    })
+    try:
+        json = res.json()
+    except ValueError as e:
+        res.raise_for_status()
+        raise ValueError(
+            'Unexpected structure of the GitHub API response: {}'.format(e)
+        )
+    else:
```

Comment:
Reading of the code was more straightforward this way for me, and also I've been changing these lines many times (alongside with testing them...), so it might as well be just a leftover. I have no strong opinions about it, I'm completely okay to remove the `else`.